ci/infra/azure/terraform.tfvars.example

```diff
@@ -0,0 +1,117 @@
+# AZURE location
+# A list of location names can be found here:
+# https://azure.microsoft.com/en-in/global-infrastructure/locations/
+# https://azure.microsoft.com/en-us/global-infrastructure/regions/
+# https://docs.microsoft.com/en-us/azure/availability-zones/az-region
+# azure_location = "West Europe"
+azure_location = "East Asia"
+
+# Enable zone for available regions
+enable_zone = false
+
+# CIRD blocks for virtual network
+cidr_block = "10.1.0.0/16"
+
+# CIDR blocks for the bastion host subnet
+bastionhost_subnet_cidr = "10.1.1.0/24"
+
+# Private subnet of virtual_network
+private_subnet_cidr = "10.1.4.0/24"
+
+# Identifier to make all your resources unique and avoid clashes with other users of this terraform project
+# stack_name = "my-sles-cluster"
+
+# Name of DNS domain
+# dnsdomain = "my.domain.com"
+dnsdomain = "my-cluster-101.example.com"
+
+# Enables creation of bastion host
+# https://docs.microsoft.com/en-us/azure/bastion/bastion-overview
+# create_bastionhost = true
+create_bastionhost = false
+
+# Number of master nodes
+# masters = 1
+masters = 1
+
+# Size of the master nodes
+# https://docs.microsoft.com/en-us/azure/virtual-machines/linux/generation-2
+# EXAMPLE:
+# master_vm_size = "Standard_D2s_v3"
+master_vm_size = "Standard_D2s_v3"
+
+# Storage account type  of the master nodes
+# https://docs.microsoft.com/en-us/azure/virtual-machines/linux/disks-types
+# EXAMPLE:
+master_storage_account_type = "StandardSSD_LRS"
+
+# "Size of the EBS volume, in Gb"
+master_disk_size = 30
+
+# Number of worker nodes
+# workers = 3
+workers = 2
+
+# Size of the worker nodes
+# EXAMPLE:
+# worker_vm_size = "Standard_D2s_v3"
+worker_vm_size = "Standard_D2s_v3"
+
+# Storage account type  of the worker nodes
+# EXAMPLE:
+# worker_storage_account_type = "Premium_LRS"
+worker_storage_account_type = "StandardSSD_LRS"
+
+# "Size of the EBS volume, in Gb"
+worker_disk_size = 30
+
+# Username for the cluster nodes
+# EXAMPLE:
+username = "sles"
+
+# Password for the cluster nodes
+# EXAMPLE:
+password = "Linux12345"
+
+# define the repositories to use
+# EXAMPLE:
+# repositories = {
+#   repository1 = "http://example.my.repo.com/repository1/"
+#   repository2 = "http://example.my.repo.com/repository2/"
+# }
+# repositories = {}
+# lb_repositories = {}
+
+# Minimum required packages. Do not remove them.
+# Feel free to add more packages
+packages = []
+
+# ssh keys to inject into all the nodes
+# EXAMPLE:
+# authorized_keys = [
+#  "ssh-rsa <key-content>"
+# ]
+authorized_keys = [
+  ""
+]
```

Comment:
I would be consistent with what has been done some lines above and have:

```
# authorized_keys = []
```

Reply:
changed